virttest/utils_disk.py

```diff
@@ -67,7 +73,7 @@ def copytree(src, dst, overwrite=True, ignore=''):
 
 
 def is_mount(src, dst=None, fstype=None, options=None, verbose=False,
-             session=None):
+             session=None, fstype_mtab=None):
```

Comment:
Could you please explain why it has to have an unused argument?

Reply:
in utils_misc.py we have the same unused argument, but the testcases are calling with it. what shall we do ?

https://github.com/avocado-framework/avocado-vt/blob/a7a6222fb267cad90854627ecde1fe57a9e20c44/virttest/utils_misc.py#L1057-L1058